services/web/server/src/simcore_service_webserver/rest.py

```diff
@@ -55,13 +58,23 @@ def setup_rest(app: web.Application):
             f"__version__.api_version_prefix {api_version_prefix} does not fit openapi.yml version {specs.info.version}"
         )
 
-    # diagnostics routes
-    routes = rest_routes.create(specs)
-    app.router.add_routes(routes)
+    # basic routes
+    app.add_routes(rest_handlers.routes)
+    if not is_diagnostics_enabled:
+        # NOTE: the healthcheck route is normally in diagnostics, but
+        # if disabled, this plugin adds a simple version of it
```

Comment:
Question: would it have been more complex to completely separate it from the diagnostics so that when you disable the diagnostics the healtchcheck is still enabled?

Reply:
@GitHK  If i understand well your question, I think the implementation will be more complex (for the same functionality) but probably it will be the right way to go since the rest plugin should know nothing about the diagnostics (but the diagnostics could know about the rest though).

I definitely should change that change but if you do not mind, due to the urgency of this PR, I would prefer to do it in a separate PR. I let you decide.